atm.js

```diff
@@ -1,3 +1,142 @@
-$(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+//set up click responses for button clicks
+// write 4 functions check withdraw/deposit / savings widthdraw/deposit
+//each function should take in both the number they typed in and the current balance
+
+
+
+
+
+
+// ------- refactoring   -------
+// single function to update balance anytime a button is pressed
+// function updateBalance(){
+//    curentballance = parseInt($(this).siblings('.balance').html().substring(1))
+//    inputVal = parseInt($(this).siblings('.input').val())
+
+
+//    }
+//change the number to red if its 0;
+// function checkforZero(){
+
+// }
+// function Overdraft
+// function makeRed(){
+//  genballance = parseInt($('.balance').text())
+//  if (genballance == 0)
+//  	(this).parent().addClass('zero')
+
+// }
+
+// ---------- original -----------------
+
+$(document).ready(function() {
+    console.log('jquery loaded');
+
+    $('#checkD').click(function() {
+        updateCheckBalance();
+    })
+    $('#checkW').click(function() {
+        withdrawChecking();
+    })
+    $('#savingsD').click(function() {
+        updateSavingsBalance();
+    })
+    $('#savingsW').click(function() {
+        withdrawSavings();
+    })
+    $(':button').click(function() {
+        zeroCheck();
+    })
+
+
+
 });
+// function takefromOther(){
+	// if this.withdrawValue > 
+//}
+
+
+function zeroCheck() {
+    cbal = parseInt($('#checkbal').text())
+    sbal = parseInt($('#savebal').text())
+        // console.log(cbal)
+
+    if (cbal === 0)
+        $('#checkbal').parent().addClass('zero')
+    if (sbal === 0)
+        $('#savebal').parent().addClass('zero')
+    if (cbal > 0)
+    	$('#checkbal').parent().removeClass('zero')
+    if (sbal > 0)
+    	$('#savebal').parent().removeClass('zero')
+}
+
+
+function withdrawChecking() {
+    ballance = parseInt($('#checkbal').text())
+    valtosub = parseInt($('#inputCheck').val())
+    if (valtosub > ballance)
+        alert('you cannot take out that much');
+    else
+        updated = (ballance - valtosub)
+    $('#checkbal').text(updated);
+}
+
+function withdrawSavings() {
+    ballance = parseInt($('#savebal').text())
+    valtoadd = parseInt($('#inputSaving').val())
+    if (valtoadd > ballance)
+        alert('you cannot take out that much');
+    else
+        updated = (ballance - valtoadd);
+    $('#savebal').text(updated);
+}
```

Comment:
As you probably noticed, the above two functions look pretty similar. How could you consolidate them into a single function? Perhaps you could make the balance and value arguments that vary depending on whether it's a checking or savings account...